### PR TITLE
Add inject course build script

### PR DESCRIPTION
**Pull request by P-Gill97**

### CODE REVIEW

File: backend/scripts/inject_course_build_sqs.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    )
+
+
+def main():
+    parser = argparse.ArgumentParser(description="")
+    parser.add_argument("course_build_id", type=str, help="Course build id")
+    args = parser.parse_args()
+    course_build_id = args.course_build_id
+
+    sqs_client = boto3.client("sqs")
```
Comment (rnathuji) [inline backend/scripts/inject_course_build_sqs.py:25]: Let's re-use the helper function [here](https://github.com/openstax/rope/blob/ba1ee9561aaf7746e4409a351f1482abd2156e69/backend/api/utils.py#L33). here.

File: backend/scripts/inject_course_build_sqs.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+from rope.api import settings
+import boto3
+
+
+def inject_course_build(sqs_client, sqs_queue_name, course_build_id):
+    queue_url_data = sqs_client.get_queue_url(QueueName=sqs_queue_name)
+    queue_url = queue_url_data["QueueUrl"]
+    message_body = {
+        "course_build_id": course_build_id
+    }
```
Comment (rnathuji) [inline backend/scripts/inject_course_build_sqs.py:12]: Let's re-use the helper function [here](https://github.com/openstax/rope/blob/ba1ee9561aaf7746e4409a351f1482abd2156e69/backend/api/utils.py#L37) for this.

Comment (P-Gill97) [reply backend/scripts/inject_course_build_sqs.py]: Updated code to use helper function.

File: backend/scripts/inject_course_build_sqs.py (hunk 1 above)
Comment (P-Gill97) [reply backend/scripts/inject_course_build_sqs.py]: Updated code to use helper function.

**Review by rnathuji — APPROVED** [review]

**Pull request merged**